readiness-tool/background.js

```diff
@@ -0,0 +1,200 @@
+/**
+ * @fileoverview background.js provides a persistant background for the extension
+ * which individual extension instances that run on different tabs use to communicate.
+ */
+
+var globals = {};
+globals.tabToUrl = {};
+
+/**
+ * Send a message to page to do parse and process the DOM.
+ * @param {Tab} tab - Tab that needs to be processed
+ */
+function handleTab(tab) {
+
+  /**
+   * Delay everything by 2seconds, so that
+   * we are sure that javascript is ready to go.
+   */
+  setTimeout(callFn, 2000)
+
+  function callFn() {
+    chrome.tabs.sendMessage(tab.id, {
+      action: 'handleTab',
+      tabId: tab.id
+    })
+  }
+
+};
+
+/**
+ * Send a message to display a loading message in the extension
+ * @param {Tab} tab - Tab that needs to be processed
+ */
+function displayLoadingInDOM(tab) {
+
+  chrome.runtime.sendMessage({
+    action: 'displayLoading',
+    tabId: tab.id
+  })
```

Comment:
lets stay consistent and not rely on automatic semi colon insertion

Reply:
Hi, Can you be a little more specific? I see no semi colons in the 4 line context I see on github (lines 36-39)

Reply:
so in these 4 lines we need to add the semi colon at the end of the statement.

instead of: